build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -37,14 +39,11 @@ public class CompileTargets
             { "debian.8-x64", "debian.8-x64" }
         };
 
-
-        private static string DotnetHostBaseName => $"dotnet{Constants.ExeSuffix}";
-        private static string DotnetHostFxrBaseName => $"{Constants.DynamicLibPrefix}hostfxr{Constants.DynamicLibSuffix}";
-        private static string HostPolicyBaseName => $"{Constants.DynamicLibPrefix}hostpolicy{Constants.DynamicLibSuffix}";
-
         [Target(nameof(PrepareTargets.Init), 
             nameof(CompileCoreHost), 
-            nameof(PackagePkgProjects))]
+            nameof(PackagePkgProjects),
+            nameof(RestoreLockedCoreHost),
```

Comment:
When we were publishing the shared framework in the cli build it made sense. Now the cli build uses the layout produced in the host build and eventually the zips.